### ISSUE TITLE
Apache Pulsar Publish only, not subscribe

### ISSUE THREAD

**Issue by whuanle** [dapr/components-contrib#1544]
### Publish only, not subscribe

Apache Pulsar component: 
```yaml
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: pulsar-pubsub
  namespace: default
spec:
  type: pubsub.pulsar
  version: v1
  metadata:
  - name: host
    value: "192.168.0.238:6650"
  - name: enableTLS
    value: "false"
```
The component is normal:
![image](https://user-images.githubusercontent.com/2189761/156087105-809a8624-6ea2-4264-b3d5-2b366ecfa69c.png)

The client pushed the message:
![image](https://user-images.githubusercontent.com/2189761/156087690-c102ca8b-277a-4966-aa44-49318077a56a.png)
![image](https://user-images.githubusercontent.com/2189761/156087479-c04558d5-5e22-4392-b0e7-baafa3558d6c.png)

My client has subscribed to the message:
![image](https://user-images.githubusercontent.com/2189761/156087822-969cf96c-f3a2-4c76-939a-843c5eb33fde.png)
![image](https://user-images.githubusercontent.com/2189761/156087852-38ef84c9-5ddb-4b08-abc1-ebcc117fdec9.png)

My dapr client did not print any logs, nor did the Web application receive any requests, but Pulsar showed that the topic had been consumed.
![image](https://user-images.githubusercontent.com/2189761/156088191-5a85ce97-4adc-404d-9945-3187d1025f2f.png)
![image](https://user-images.githubusercontent.com/2189761/156088559-2176e4b0-47c6-4a9e-9e29-92fce18f706f.png)
![image](https://user-images.githubusercontent.com/2189761/156088632-5934ecb4-838d-4a70-8a35-73c58d7d2356.png)

### Normal message queue components

I tested Pulsar、RabbitMQ、Redis:
![image](https://user-images.githubusercontent.com/2189761/156088852-0a8c8493-b1b0-46b8-be5e-b4ca27eec1aa.png)

![image](https://user-images.githubusercontent.com/2189761/156088918-7de06baa-f086-49a1-840b-48ffd35c4fed.png)
![image](https://user-images.githubusercontent.com/2189761/156088951-56feabcc-9101-4faa-b7a5-48f2ef326bda.png)

![image](https://user-images.githubusercontent.com/2189761/156089168-455a8a9a-46a2-4c4d-a504-03ba4b72fbe0.png)
RabbitMQ and Redis are fine, but Pulsar is not.

**Comment by whuanle** [comment]
This is fine if you write code to connect to Pulsar directly.
![image](https://user-images.githubusercontent.com/2189761/156090317-1adfe53a-2912-40e6-85ae-e6f5ca4232ef.png)

![image](https://user-images.githubusercontent.com/2189761/156090287-61d64d6a-4b78-460e-b0c6-f53694cc79eb.png)

**Comment by justinwujian** [comment]
I also had this problem and it made me wonder

**Comment by whuanle** [comment]
> I also had this problem and it made me wonder

@justinwujian 
The problem has been solved.The Master branch has been resolved, but no new version has been released.
release/tag 1.6.0 ×
master                 √

**Comment by shubham1172** [comment]
@whuanle, @justinwujian yes, there were some fixes for pulsar after 1.6.0. Can you please try with the latest tag v1.7.0-rc.4?

**Comment by whuanle** [comment]
@shubham1172 
This issue has been resolved.
After testing, it is fully available.
